rosdep/base.yaml

```diff
@@ -6195,3 +6223,4 @@ zziplib:
   opensuse: [zziplib-devel]
   rhel: [zziplib-devel]
   ubuntu: [libzzip-0-13, libzzip-dev]
+
```

Comment:
Really minor nit, but please remove this trailing whitespace.  I'm surprised it didn't trip our linters.

Reply:
oh, sorry. I remove the trailing whitespace.